smf/svc-kayak

```diff
@@ -130,7 +130,7 @@ class KayakLog(object):
             raise cherrypy.HTTPError("404 Not Found")
         log = open(os.path.join(options.basedir, 'log',
             macaddr + "." + str(time.time())), 'w')
-        log.write(cherrypy.request.body.read())
+        log.write(str(cherrypy.request.body.read()))
```

Comment:
When would you not get a string? Just blindly turning everything into a string seems a tad risky ... as it might provide the opportunity to play fun games with the terminal where someone looks at the log file ... I don't know enough about the code to judge though ...